tests/impulse-tests/Makefile

```diff
@@ -122,9 +122,8 @@ travis:
 #########################################################################
 # automatic github action test
 github_action:
+	$(MAKE) -f Make.rust outdir=rust/inpl CARGOOPTIONS="" FAUSTOPTIONS="-I ../../libraries/ -double -inpl -rnt -a archs/rust/architecture_inpl.rs"
 	$(MAKE) -f Make.rust outdir=rust/no CARGOOPTIONS="" FAUSTOPTIONS="-I ../../libraries/ -double -a archs/rust/architecture_trait.rs"
-	$(MAKE) -f Make.rust outdir=rust/cm CARGOOPTIONS="" FAUSTOPTIONS="-I ../../libraries/ -double -cm -a archs/rust/architecture_cm.rs"
-	$(MAKE) -f Make.rust outdir=rust/ecrnt CARGOOPTIONS="" FAUSTOPTIONS="-I ../../libraries/ -double -ec -rnt -a archs/rust/architecture_ecrnt.rs"
```

Comment:
because they didn't change but they changed in the previous pr so i added them.
the idea is to have two basis testssets on github. one for rust and one for cpp and additional tests that are affected by the change or new.

Reply:
That's an unusual strategy to do testing. Often knowing _that nothing has changed_ is exactly the hard part, and having them executed automatically on the CI answers that. And also whether any future changes would break anything here.

In the end it's up to @sletz how we want to handle that in case the CI times are getting too long.

Reply:
Well, no definitive answer here: testing all backends with all of their options are out of reach for CI. So we could keep a light _github_action_ with 2,3 for Rust and 1,2 for C++. In any case I usually rust most of the test locally when I make heavy changes.